practice-app/app/views.py

```diff
@@ -239,3 +241,64 @@ def WorldTime():
                 languageCode=world_time.languageCode,
             )
         return render_template("worldtime.html")
+
+
+@app.route(
+    "/bored", methods=["GET"]
+)  # it is a decorator we have to put a function under of it
+@login_required
+def Bored():
+    if request.method == "GET":
+        response = requests.get("http://www.boredapi.com/api/activity/")
+
+        if response.status_code != 200:
+            return render_template("bored.html", error=response.status_code)
+
+        response = response.json()
+        return render_template("bored.html", kamela=response)
+    else:
+        return render_template("bored.html", error=405)
+
+
+@app.route(
+    "/bored/save", methods=["POST"]
+)  # it is a decorator we have to put a function under of it
+@login_required
+def BoredSave():
+    if request.method == "POST":
+        activityName = request.form.get("activity")
+
+        session.add(Activities(activity_name=activityName, user_id=current_user.id))
+        session.commit()
+
+        return "SAVED!", 201
```

Comment:
Make this redirect instread of showing text.

Reply:
> Is there a specific reason of returning status code in methods? If not, we can remove them since it is working without them.

We are not only supposed to handle the status codes when using the external APIs, but also while returning responses to the client. The instructor said so, and there is no harm in returning status codes anyways, IMHO.

Reply:
> Make this redirect instread of showing text.

I implemented redirection.